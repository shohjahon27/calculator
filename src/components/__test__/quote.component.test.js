import Enzyme, { shallow } from "enzyme";

import Adapter from "enzyme-adapter-react-16";
import QuoteComponent from "../quote.component";
import toJson from "enzyme-to-json";

Enzyme.configure({ adapter: new Adapter() });

describe("Quote Component", () => {
  it("should render correctly", () => {
    const wrapper = shallow(<QuoteComponent />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
