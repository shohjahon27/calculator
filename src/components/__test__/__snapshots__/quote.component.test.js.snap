// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Component should render correctly 1`] = `
<WithStyles(ForwardRef(Card))
  className="makeStyles-card-1"
>
  <img
    alt="Beauty of mathematics"
    src="qoute.jpg"
  />
</WithStyles(ForwardRef(Card))>
`;
